src/components/Account/Account.tsx

```diff
@@ -65,6 +66,9 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
     shallow,
   );
 
+  // refetch user data on visit (this is only JW integration case)
+  const { isLoading } = useQuery(['userInfo'], () => getUserInfo());
```

Comment:
This should be moved up one level. Stores/controllers should only be used in pages and containers.